ckanext/ioos_theme/controllers/feedback.py

```diff
@@ -26,43 +27,42 @@ def index(self, data=None, errors=None, error_summary=None):
                             will be passed to the controller
         :param dict error_summary: Summary of any validation errors
         '''
+        name = ""
+        email = ""
+        feedback = ""
         # If the HTTP request is POST
         if request.params:
-            return self._post_feedback()
+            try:
+                if request.params['g-recaptcha-response']:
+                    return self._post_feedback()
+                else:
+                    name = request.params['name']
+                    email = request.params['email']
+                    feedback = request.params['feedback']
+                    h.flash_notice(_('Must enter captcha below'))
+            except KeyError:
+                name = request.params['name']
+                email = request.params['email']
+                feedback = request.params['feedback']
+                h.flash_notice(_('Must enter captcha below'))
 
-        data = data or {}
+        data = data or {"name": "", "email": "", "feedback": ""}
+        data['name'] = name or ""
+        data['email'] = email or ""
+        data['feedback'] = feedback or ""
         errors = errors or {}
         error_summary = error_summary or {}
-        vars = {
-            'package_name': None,
-            'data': data,
-            'errors': errors,
-            'error_summary': error_summary
-        }
-        return render('feedback/form.html', extra_vars=vars)
-
-    def feedback_package(self, package_name=None, data=None, errors=None,
-                         error_summary=None):
-        '''
-        Returns a render for the feedback form.
+        site_key = config.get('feedback.site_key') or ""
 
-        :param dict data: Unused
-        :param dict errors: Any validation errors that the user has entered
-                            will be passed to the controller
-        :param dict error_summary: Summary of any validation errors
-        '''
-        # If the HTTP request is POST
-        if request.params:
-            return self._post_feedback()
+        if not site_key:
+            logging.warning('Administrator must setup feedback.site_key')
 
-        data = data or {}
-        errors = errors or {}
-        error_summary = error_summary or {}
         vars = {
```

Comment:
Keep in mind `vars` is a Python builtin function, and while it can be overridden, it might potentially confuse in some contexts.  Don't worry about it now, but keep in mind in the future.